pki.spec

```diff
@@ -553,6 +554,7 @@ Requires:         apache-commons-io
 Requires:         apache-commons-lang
 Requires:         apache-commons-logging
 Requires:         jakarta-commons-httpclient
+Requires:         jackson-dataformat-xml
```

Comment:
Sorry for picking on WIP commits, but what's this for? If this is (truly) required, just noting that we'll have to pick this up in the SIG and it isn't part of the other packages we knew we'd have to pick up...  :)

Reply:
As discussed on IRC, I was trying to create an XML configuration for the ACME service, but let me investigate to see if we can use existing dependencies or just use plain Java properties.